.changeset/tricky-horses-begin.md

```diff
@@ -0,0 +1,5 @@
+---
+'@nordeck/matrix-neoboard-widget': minor
+---
+
+Change the design of the neoboard info dialog
```

Comment:
The about dialog was not released yet, so https://github.com/nordeck/matrix-neoboard/blob/137a943d5c686c48183ed1816f9d0b727d2a96fa/.changeset/famous-suits-judge.md would still be valid and we don't need a second changeset for it.